pkg/apis/hive/v1/gcp/machinepools.go

```diff
@@ -9,6 +9,20 @@ type MachinePool struct {
 	// eg. n1-standard-4
 	InstanceType string `json:"type"`
 
+	// ControlPlaneSubnet defines the subnet where the control plane will be deployed.
+	// The value should be the name of the subnet.
+	// If ommitted one will be created by the installer.
+	ControlPlaneSubnet string `json:"control_plane_subnet,omitempty"`
+
+	// ComputeSubnet defines the subnet where the compute nodes will be deployed.
+	// The value should be the name of the subnet.
+	// If ommitted one will be created by the installer.
```

Comment:
the doc comment seems not valid in the context of hive.

Reply:
Should I remove the last sentence? what would you suggest?